src/pages/workspace/WorkspaceMoreFeaturesPage.tsx

```diff
@@ -219,4 +220,4 @@ function WorkspaceMoreFeaturesPage({policy, route}: WorkspaceMoreFeaturesPagePro
 
 WorkspaceMoreFeaturesPage.displayName = 'WorkspaceMoreFeaturesPage';
 
-export default withPolicyAndFullscreenLoading(WorkspaceMoreFeaturesPage);
+export default withPolicyConnections(WorkspaceMoreFeaturesPage);
```

Comment:
This is causing a bug.
1. Sign out
2. Sign in
3. Go offline
4. Go to the more features page
5. Page should be accessible offline

Reply:
I just realised that all the components that use this HOC do not work offline (unless the connections are already present in Onyx)

Reply:
@aldo-expensify, how should be handle this? We want to make sure that we don't allow the admin from disabling the accounting feature when the workspace is connected to the QBO. But to do that, we need the `connections` data. When the `connections` data hasn't been fetched and the device is offline, we cannot tell if there is an accounting connection or not.

Reply:
@s77rt, but all screens, including Categories and Tags screens, that were previously offline first that were later modified to use `withPolicyConnections` will not be offline first

Reply:
Yes that's correct.